src/components/structures/MatrixChat.js

```diff
@@ -1061,6 +1061,7 @@ module.exports = React.createClass({
                     defaultIsUrl={this.getDefaultIsUrl()}
                     brand={this.props.config.brand}
                     teamServerConfig={this.props.config.teamServerConfig}
+                    queryParams={this.props.startingFragmentQueryParams}
```

Comment:
If we're just extracting one thing from this, I'd prefer to extract it in MatrixChat and pass the actual thing in like we do for email. That or we should just pass the query params in and have Registration extract everything, but ideally query params should be handled by the containing web page, not the components, so failing that I'd rather keep them confined to MatrixChat.